File: packages/kevin_flutter_in_app_payments/kevin_flutter_in_app_payments_ios/ios/Classes/Entity/PaymentSessionConfigurationEntity.swift
import Foundation

internal struct PaymentSessionConfigurationEntity : Codable {
    let paymentId: String
    let preselectedCountry: String?
    let disableCountrySelection: Bool
    let countryFilter: [String]
    let bankFilter: [String]
    let preselectedBank: String?
    let skipBankSelection: Bool
    let skipAuthentication: Bool
}

File: packages/kevin_flutter_in_app_payments/kevin_flutter_in_app_payments_android/android/src/main/kotlin/eu/kevin/flutter/payments/entity/PaymentSessionConfigurationEntity.kt
package eu.kevin.flutter.payments.entity

import kotlinx.serialization.Serializable

@Serializable
internal data class PaymentSessionConfigurationEntity(
    val paymentId: String,
    val preselectedCountry: String?,
    val disableCountrySelection: Boolean,
    val countryFilter: List<String>,
    val bankFilter: List<String>,
    val preselectedBank: String?,
    val skipBankSelection: Boolean,
    val skipAuthentication: Boolean,
)